cypress/integration/t2.js

```diff
@@ -0,0 +1,68 @@
+import LoginPage from '../objects/loginPage';
+
+const loginPage = new LoginPage();
+
+it('Should verify all user roles can log in and should see appropriate tabs - USER', function () {
```

Comment:
I think these test should be in `login.js` file as they check login functionality